bifrost/proxy/pages/passthru/+config.ts

```diff
@@ -2,9 +2,8 @@ import { Config } from "vike/types";
 
 export default {
   route: "/*",
+  clientRouting: false,
```

Comment:
this is the key. 
The requirement for passthru page is that it exists in the eye of the Vike router, but does a full page reload to get the proxied page, with no extra bifrost stuff.

Previously we were enforcing the passthru routes to fetch the page from the server by calling `location.reload()` in onRenderClient. For reasons still not entirely clear to me, that caused this bug.

Setting `clientRouting: false` tells Vike to just load the page without any SPA features, which is what we want.